site/src/_pages/docs/4_faq.mdx

```diff
@@ -0,0 +1,134 @@
+
+You can find all [Schema.org](http://Schema.org) definitions on the Block Protocol website already, ready to be ‘crosswalked’ back to.
+
+</InfoCardWrapper>
+
+### Why does the Block Protocol include a schema editor?
+
+Schemas are central to our vision for the Block Protocol.
+
+We don’t believe that a single set of schemas provided by any one standards organization (even schema.org) can ever perfectly fit all use cases, and as such the ability to create new schemas is important. When doing this, we want to make them as accessible to machines (and people) as possible.
+
+<InfoCardWrapper>
+
+The Block Protocol website includes a schema editor that provides a convenient way to define new entity types used by or with blocks and their embedding applications. These are then hosted persistently and made accessible via the same content delivery network that serves blocks from the Block Protocol Hub, guaranteeing their availability and discoverability.
+
+<InfoCard variant="warning" title="Who can see my schemas?">
+
+All schemas created on the Block Protocol website are currently public. In future we’ll support the creation of private schemas as well.
+
+</InfoCard>
+
+</InfoCardWrapper>
+
+### Why aren’t the default [Schema.org](http://Schema.org) definitions sufficient?
+
+[Schema.org](http://Schema.org) provides a great base ontology for defining lots of types of ‘things’ out there in the world.
+
+1.  Many times, certain ‘properties’ or data won’t be relevant to your use case, resulting in bloated entity type definitions.
+1.  On other occasions you’ll want to store information differently than how it’s set out in the [schema.org](http://schema.org) definition. For example, [schema.org/Person](http://schema.org/Person) defines a person as having a `givenName` and a `familyName` and yet in various cultures this isn’t guaranteed. HASH uses `preferredName` and `legalName` instead - in communications and billing contexts respectively. You can view the HASH ‘Person’ schema at [blockprotocol.org/types/XXXXX](http://blockprotocol.org/types/XXXXX) and see how it crosswalks with the canonical definition of a person provided by Schema.org. This is just one example of how custom schemas can be made to relate back to the core ontology provided by Schema.org.
+1.  The [Schema.org](http://Schema.org) ontology is slow to change. This is by design, similar to how most standards organizations operate. And while it can move fast when clear universal impetus exists (evidenced by its excellent response to COVID) it ultimately represents a pseudo-centralized model of maintaining a schema registry. The Block Protocol provides the permissionless ability to build atop and extend [schema.org](http://schema.org), unbounded by the processing constraints and imagination of any single working group or organization.
```

Comment:
```suggestion
1.  The [schema.org](https://schema.org) ontology is slow to change. This is by design, similar to how most standards organizations operate. And while it can move fast when clear universal impetus exists (evidenced by its excellent response to COVID) it ultimately represents a pseudo-centralized model of maintaining a schema registry. The Block Protocol provides the permissionless ability to build atop and extend [schema.org](https://schema.org), unbounded by the processing constraints and imagination of any single working group or organization.
```